src/examples/using-with-cmake/ascent_render_example.cpp

```diff
@@ -48,11 +48,10 @@
 ///
 //-----------------------------------------------------------------------------
 
-#include <iostream>
-
-#include "ascent.hpp"
+#include <ascent/ascent.hpp>
+#include <conduit/conduit_blueprint.hpp>
```

Comment:
are those the official includes?

Reply:
internally, parts of the libs don't include using the prefix style (conduit/conduit_more_specific_header_zzz.hxx)

so for installs, we provide both styles of include paths either via cmake or our config.mk, both styles being a setup that enables both:

#include <conduit_blueprint.hpp>
#include <conduit/conduit_blueprint.hpp>

All of the headers are also prefixed with ascent_ (or conduit_), so they wont collide with other products (restated: nothing like utils.hxx exists to collide)

Reply:
Sounds good. Since this is an example where people copy-paste from and since there is already a modern prefix dir (even though there is also a prefix in each name) I'll keep it at `<ascent/ascent.hpp>` and similar for conduit :)